packages/@react-native-windows/cli/src/test/dependencyConfig.test.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+
+import {
+  dependencyConfigWindows,
+  WindowsDependencyConfig,
+} from '../config/dependencyConfig';
+
+import {copyAndReplace} from '../generator-common';
+
+const templateRoot = path.resolve('../../../vnext/template');
+
+const testProjectGuid = '{416476D5-974A-4EE2-8145-4E331297247E}';
+
+function projectTests(testLibName: string, setup?: Function): void {
+  const folder = path.resolve('src/test/projects/', testLibName);
+  const rnc = require(path.join(folder, 'react-native.config.js'));
+
+  if (setup !== undefined) {
+    setup(folder);
+  }
+
+  test(`dependencyConfig - ${testLibName} (userConfig is null)`, () => {
```

Comment:
I think this is fine, but a tip for the future is that Jest has builtin support for this pattern by using `test.each`. See https://github.com/microsoft/react-native-windows/blob/master/packages/react-native-platform-override/src/test/Override.test.ts#L99-L102 for an example.

Reply:
Fixed.